charts/orchestrator/templates/tekton-tasks.yaml

```diff
@@ -207,6 +207,7 @@ spec:
       image: registry.access.redhat.com/ubi9-minimal
       workingDir: $(workspaces.workflow-source.path)/flat/$(params.workflowId)
       script: |
+        echo "quarkus.flyway.migrate-at-start=true" >> application.properties
```

Comment:
We should make this optional somewhere, somehow.

Reply:
I'll see how to do that